TROUBLESHOOTING.md

```diff
@@ -1,17 +1,19 @@
 # Troubleshooting Guide
 
-## Issues starting MySQL or PostgreSQL with docker-compose
```

Comment:
I'm definitely open to updating the troubleshooting guide, but not in an otherwise-unrelated PR. Would you mind removing these changes from this PR and opening a separate PR just for them?

Reply:
I should've already had these changes merged as part of a previous PR I created - I am going to sync my fork, merge, and then resubmit this PR to try and get rid of these edits